lib/matplotlib/tests/test_figure.py

```diff
@@ -557,6 +557,26 @@ def test_savefig_pixel_ratio(backend):
     assert ratio1 == ratio2
 
 
+@image_comparison(['transparent_inset_axes'],
+                    extensions=['png'], savefig_kwarg={'transparent': True})
```

Comment:
- please use `remove_text=True` to remove the ticklabels (this makes the images more robust against underlying changes of the library.
- please combine the two tests together (by putting the inset_axes into an axes in a subfigure).

Other than that, this looks good to me.

Reply:
(don't forget to remove the now unneeded images)

Reply:
Done!